SecurityAgent/SecurityAgent.json

```diff
@@ -247,7 +246,7 @@
                 "type": "object",
                 "properties": {
                     "token": {
-                        "description": "Signed JsonWeb token",
+                        "description": "On success, returns Signed JsonWeb token. On failure, returns error message and error code from the following table.",
```

Comment:
this "description" field is for the "token" property of "result". Neither this property nor "result" return "error message and error code". On error, "error" object is set instead of "result". Explanation of what the method returns should probably be in the method description.

Reply:
Modified the document as per your suggestion